plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/AppEngineFlexFacet.java

```diff
@@ -29,14 +29,18 @@
   public static final String ID = "com.google.cloud.tools.eclipse.appengine.facets.flex";
   public static final String VERSION = "1";
 
+  public static final IProjectFacet FACET = ProjectFacetsManager.getProjectFacet(ID);
+  public static final IProjectFacetVersion FACET_VERSION = FACET.getVersion(VERSION);
```

Comment:
Here it's `VERSION` whereas in `AppEngineStandardFacet`, it's `.getDefaultVersion()`.